control/tests/timebase_test.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+import inspect
+import numpy as np
+import control as ct
+
+@pytest.mark.parametrize(
+    "dt1, dt2, dt3", [
+        (0, 0, 0),
+        (0, 0.1, ValueError),
+        (0, None, 0),
+        (0.1, 0, ValueError),
+        (0.1, 0.1, 0.1),
+        (0.1, None, 0.1),
+        (None, 0, 0),
+        (None, 0.1, 0.1),
+        (None, None, None),
+        (0.2, None, 0.2),
+        (0.2, 0.1, ValueError),
```

Comment:
Great PR. Would be good to add tests for `dt=True`, which are compatible with all other discrete-time timebases. https://python-control.readthedocs.io/en/0.9.2/conventions.html

Reply:
Great suggestion. Done.